contracts/deploy/020_aave_v2.js

```diff
@@ -0,0 +1,102 @@
+const { deploymentWithProposal } = require("../utils/deploy");
+
+module.exports = deploymentWithProposal(
+  { deployName: "020_aave_v2" },
+  async ({
+    assetAddresses,
+    deployWithConfirmation,
+    ethers,
+    getTxOpts,
+    withConfirmation,
+  }) => {
+    const { deployerAddr, governorAddr } = await getNamedAccounts();
+    // Signers
+    const sDeployer = await ethers.provider.getSigner(deployerAddr);
+    const sGovernor = await ethers.provider.getSigner(governorAddr);
+    // Current contracts
+    const cVaultProxy = await ethers.getContract("VaultProxy");
+    const cVaultAdmin = await ethers.getContractAt(
+      "VaultAdmin",
+      cVaultProxy.address
+    );
+    const cOldAaveStrategy = await ethers.getContract("AaveStrategyProxy");
+    // Deployer Actions
+    // ----------------
+
+    // 1. Deploy new proxy
+    // New strategy will be living at a clean address
+    const dAaveStrategyProxy = await deployWithConfirmation(
+      "AaveStrategyProxy",
+      [],
+      "InitializeGovernedUpgradeabilityProxy"
+    );
+    const cAaveStrategyProxy = await ethers.getContractAt(
+      "InitializeGovernedUpgradeabilityProxy",
+      dAaveStrategyProxy.address
+    );
+    // 2. Deploy new implementation
+    const dAaveStrategyImpl = await deployWithConfirmation("AaveStrategy");
+    const cAaveStrategy = await ethers.getContractAt(
+      "AaveStrategy",
+      dAaveStrategyProxy.address
+    );
+    // 3. Init the proxy to point at the implementation
+    await withConfirmation(
+      cAaveStrategyProxy
+        .connect(sDeployer)
+        ["initialize(address,address,bytes)"](
+          dAaveStrategyImpl.address,
+          deployerAddr,
+          []
+        )
```

Comment:
missing the getTxOpts() which is super helpful to speed up tx by bumping up gas price
same comment for the other calls to withConfirmation that don't include it in this file.

Reply:
Updated. Thanks!